end-to-end-test/local/docker/setup_docker_containers.sh

```diff
@@ -51,26 +51,13 @@ build_database_container() {
     while ! docker run --rm --net=$DOCKER_NETWORK_NAME mysql:5.7 mysqladmin ping -u $DB_USER -p$DB_PASSWORD -h$DB_HOST --silent; do
         echo Waiting for cbioportal database to initialize...
         sleeptime=$sleeptime+10
-        if (($sleeptime > $maxtime)); then 
+        if (($sleeptime > $maxtime)); then
             echo Timeout reached. Terminating test!
             exit 1
         fi
         sleep 10
     done
 
-    # when seed database has been loaded, create a container that does 
-    # not depend on the seed data to be present in /tmp (this benefits local testing only)
-    (docker stop $DB_HOST && docker rm $DB_HOST) || true
-    docker run -d \
-        --name=$DB_HOST \
-        --net=$DOCKER_NETWORK_NAME \
-        -e MYSQL_ROOT_PASSWORD=$DB_USER \
-        -e MYSQL_USER=$DB_USER \
-        -e MYSQL_PASSWORD=$DB_PASSWORD \
-        -e MYSQL_DATABASE=$DB_PORTAL_DB_NAME \
-        -v "$DB_DATA_DIR:/var/lib/mysql/" \
-        mysql:5.7
```

Comment:
@alisman is the restart slow? I guess Q is "how does this part benefit local testing"

Reply:
@inodb For some reason this causes things to fail for me.  Everything works fine without it. A while back @pvannierop said he didn't recall why this was necessary and was ok with our removing it

Reply:
No, its fine to remove this.